treeherder/intermittents_commenter/management/commands/run_intermittents_commenter.py

```diff
@@ -0,0 +1,16 @@
+from django.core.management.base import BaseCommand
+
+from treeherder.intermittents_commenter.commenter import Commenter
+
+
+class Command(BaseCommand):
+    """Management command to manually initiate intermittent failures commenter
+        default is daily and non-test mode, use flags for weekly and test mode."""
+
+    def add_arguments(self, parser):
+        parser.add_argument('-w', '--weekly', action='store_true', dest='weekly_mode', help='generate weekly, rather than daily, comment summaries')
+        parser.add_argument('--dry-run', action='store_true', dest='dry_run', help='output comments to stdout rather than submitting to Bugzilla')
+
+    def handle(self, *args, **options):
+        process = Commenter(**options)
```

Comment:
Ah so it turns out `options` contains a bunch of other stuff which we probably shouldn't pass to `Commenter`, so let's double back a bit, and change this to:

```python
process = Commenter(weekly_mode=options['weekly_mode'], dry_run=options['dry_run'])
```

(This still avoids the previous conditionals at least :-))